src/sonic-config-engine/minigraph.py

```diff
@@ -200,7 +200,17 @@ def parse_dpg(dpg, hname):
                     acl_intfs = port_alias_map.values()
                     break;
             if acl_intfs:
-                acls[aclname] = { 'policy_desc': aclname, 'ports': acl_intfs, 'type': 'MIRROR' if is_mirror else 'L3'}
+                acls[aclname] = {'policy_desc': aclname,
+                                 'ports': acl_intfs,
+                                 'type': 'MIRROR' if is_mirror else 'L3',
+                                 'service': 'N/A'}
```

Comment:
So is this "service" attribute only feasible for CTRLPLANE type? Will orchagent read this field for L3 and MIRROR?

Reply:
It is only applicable to CTRLPLANE type. Orchagent currently has no concept of this field, so it is effectively ignored for L3 and MIRROR ACLs.